docs/src/sway-program-types/scripts.md

```diff
@@ -1,10 +1,10 @@
 # Scripts
 
-A script is deployed bytecode on the chain which executes once to perform some task. It does not represent ownership of any resources and it cannot be called by a contract. A script can return a single value of any type.
+A script is runnable bytecode on the chain which executes once to perform some task. It does not represent ownership of any resources and it cannot be called by a contract. A script can return a single value of any type.
 
 Scripts are state-aware in that while they have no persistent storage (because they only exist during the transaction) they can call contracts and act based upon the returned values and results.
 
-Under the hood, the Rust and TypeScript SDK wrap all smart contract calls from EOAs with scripts that contain minimal code to make the call and forward script data as call parameters.
+Under the hood, the Rust and TypeScript SDKs wrap all smart contract calls from addresses with scripts that contain minimal code to make the call and forward script data as call parameters.
```

Comment:
I don't have a suggestion here but this sentence is difficult for me to understand. Maybe we could break it down into smaller sentences and provide a bit more explanation?

Reply:
Fixed in [22ccf12](https://github.com/FuelLabs/sway/pull/1029/commits/22ccf12a3e0afff6e644ef6ec7f1e4c6dba98aa5)